lib/honeybadger/agent.rb

```diff
@@ -141,6 +142,11 @@ def notify(exception_or_opts, opts = {})
         return false
       end
 
+      if notice.halted?
+        debug { sprintf('halted notice feature=notices id=%s', notice.id) }
```

Comment:
The id should be omitted from the log here, as the notice won't be submitted, making the id meaningless.